src/redux/studio-permissions.js

```diff
@@ -3,12 +3,12 @@ const {selectUserId, selectIsAdmin, selectIsSocial,
     selectHasFetchedSession, selectStudioCommentsGloballyEnabled} = require('./session');
 
 // Fine-grain selector helpers - not exported, use the higher level selectors below
-const isCreator = state => selectUserId(state) === state.studio.owner;
```

Comment:
I know this has already been merged, but shouldn't the parameter currently named as "owner" be changed to "host" to improve maintainability and readability of the code?